src/main/java/cloud/fogbow/common/util/connectivity/cloud/cloudstack/CloudStackHttpToFogbowExceptionMapper.java

```diff
@@ -18,9 +19,20 @@ public static void map(HttpResponseException e) throws FogbowException {
             case HttpStatus.SC_NOT_FOUND:
                 throw new InstanceNotFoundException(e.getMessage(), e);
             default:
-                if (e.getStatusCode() > 204) {
-                    throw new FogbowException(e.getMessage(), e);
-                }
+                throw new FogbowException(e.getMessage(), e);
         }
     }
+    
+    public static FogbowException get(HttpResponseException e) throws FogbowException {
```

Comment:
Can this method name be improved? Also, this throws clause is unnecessary.

Reply:
This name will become "map()" at the end. It will be the last task.
About the clause, I will fix it.

Reply:
Done